pems-server/src/test/java/org/maroubra/pemsserver/monitoring/utsapi/WebSensorTest.java

```diff
@@ -0,0 +1,31 @@
+package org.maroubra.pemsserver.monitoring.utsapi;
+
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.google.common.truth.Truth.assertThat;
+
+public class WebSensorTest {
+
+    @Test
+    public void getSensorData() {
+        WebSensor webSensor1 = new  WebSensor("wasp","ES_B_11_429_3E90","BAT");
```

Comment:
Ideally, with above changes for injecting `WebApiRequest` we should probably mock the api method so we don't rely on UTS for passing our builds 😆